go.mod

```diff
@@ -21,7 +21,6 @@ require (
 	golang.org/x/crypto v0.0.0-20191011191535-87dc89f01550 // indirect
 	golang.org/x/net v0.0.0-20191014212845-da9a3fd4c582 // indirect
 	golang.org/x/time v0.0.0-20190921001708-c4c64cad1fd0 // indirect
-	gopkg.in/yaml.v2 v2.2.4
```

Comment:
why did go.mod and go.sum change when there don't appear to be any source code changes?

Reply:
Running the release runs `go mod tidy`, and that made these changes